backend/app/models.py

```diff
@@ -1,6 +1,73 @@
 import json
 
-from . import db
+from . import db, http_auth
+from itsdangerous import TimedJSONWebSignatureSerializer as Serializer
+from itsdangerous import BadSignature, SignatureExpired
 from werkzeug.security import generate_password_hash, check_password_hash
-
+from sqlalchemy.dialects.postgresql import UUID
+import uuid
 # create model classes here
+
+
+#Authorization for User
+@http_auth.verify_token
+def verify_auth_token(token):
+    s = Serializer(current_app.config["SECRET_KEY"])
+    try:
+        data = s.loads(token)
+    except SignatureExpired:
+        return False  # valid token, but expired
+    except BadSignature:
+        return False  # invalid token
+    user = MuUser.query.get(data["token"])
+    return user
+#MuUser Domain Model Class
+class MuUser(db.Model)    
+    #Initializing Table
+    __tablename__='users'
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4, unique=True, nullable=False)
+    name = db.Column(db.String(64), nullable=False)
+    email = db.Column(db.String(64), unique=True, nullable=False)
+    password_hash = db.Column(db.String(128), nullable=False)
```

Comment:
TBD on what this will be since I've never used Firebase auth 🤷‍♀️ this might be a already encrypted pw